packages/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -77,7 +77,7 @@ export const Wizard = ({
   onStepChange,
   onSave,
   onClose,
-  shouldFocusContent = false,
+  shouldFocusContent = true,
```

Comment:
@tlabaj I believe we had said that for v6 we wanted to make this the default, rather than an opt in, correct?

Reply:
Yes,  that is what was discussed for accessibility purposes